.env.example

```
@@ -177,6 +177,7 @@ SEARCH_FUZZY_TOLERANCE=1
 EMAIL_FROM=Nibleaf <no-reply@example.com>
 POSTMARK_API_KEY=
 POSTMARK_MESSAGE_STREAM=
+EMAIL_DELIVERY_REQUIRED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_DELIVERY_REQUIRED key should go before the EMAIL_FROM key

(UnorderedKey)